test/dummy/app/views/home/index.html

```diff
@@ -1,8 +1,29 @@
-<h1>Hello!</h1>
-<%= copilot_text 'foobar' do %>
-  I am editable
+<ol>
+  <li>Home</li>
+  <li><%= link_to 'About', about_path %></li>
+  <li><%#= link_to 'Stuff', stuff_path %></li>
+</ol>
+
+<h1>Hello, and welcome to the Copilot dummy app!</h1>
+<h2>Here we will demonstrate Copilot in all its glory</h2>
+
+<h3><%= copilot_text 'paragraph_1_header', 'Behold a paragraph' %></h3>
+
+<p>
+<%= copilot_text 'paragraph_1_text' do %>
+The path of a cosmonaut is not an easy, triumphant march to glory. You have to get to know the meaning not just of joy but also of grief, before being allowed in the spacecraft cabin.
+
+Astronomy compels the soul to look upward, and leads us from this world to another.
+
+You know, being a test pilot isn't always the healthiest business in the world.
+
+You know, being a test pilot isn't always the healthiest business in the world.
+
+Curious that we spend more time congratulating people who have succeeded than encouraging people who have not.
 <% end %>
+</p>
+
+<%= copilot_text 'paragraph_2_header', 'And ye, another', 'h3' %>
```

Comment:
its optional, you can do either one, look at the first header ;)  From a styling perspective I thought that giving the dev more flexibility would be good.